src/common/components/ConditionalLink.js

```diff
@@ -0,0 +1,11 @@
+import { Link } from 'react-router-dom';
+
+const ConditionalLink = ({ children, to, condition }) => {
+  if (!!condition && to) {
+    return <Link to={to}>{children}</Link>;
+  } else {
+    return <>{children}</>;
```

Comment:
Question: Should this be wrapped by `Typography`? I think it will be just text directly in the markup without it.

Reply:
Not necessarily. In this case, I have an image, which doesn't need a wrapper.